### ISSUE TITLE
Does this include historical data or just current listings?

### ISSUE THREAD

**Issue by whydna** [harry-s-grewal/realtorca#10]

**Comment by simran-s-grewal** [comment]
The historical data is sometimes included in the listing itself. However that's not always true, so this will only pull whatever data is currently online.

**Comment by whydna** [comment]
awesome - any idea on size of a complete historical crawl? also, does it include photos?

tnx so much for htis.

**Comment by simran-s-grewal** [comment]
It doesn't include photos, but it includes links to photos. If you'd like to download them, it shouldn't be too hard to parse those links with a script. The complete size of the crawl will likely only be a few mb at most, since it's text only.
